v2/community/upgrading-supertokens.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: upgrading-supertokens
+title: Upgrading Supertokens core and SDKs
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./community/upgrading-supertokens.mdx -->
+
+# Upgrading SuperTokens core and SDKs
+
+:::important
+You need to follow these steps in order to properly upgrade supertokens
+:::
+
+In order to maintain compatibility and keep up on new features, we recommend upgrading your core and SDKs whenever possible.
+
+## 1. Upgrade your core
+
+You will need to shut down your core in order to upgrade it. After it's off, run the migration scripts for all major versions inbetween
+your starting version and the version you are upgrading to. If the supertokens core version you have upgraded to shows as compatible
+with your current front-end and back-end sdk's in the [compatibility table](./compatibility-table), then you can bring your core back online.
+Otherwise, please leave your core offline as you continue to upgrade your other SDK's.
```

Comment:
- doesnt make sense to tell people to leave the core offline if their backend sdk is not compatible with the core.. you need to instead tell them to keep the core online and then not start the backend until the backend sdk is upgraded.
- There are migration scripts in minor update versions as well (See changelog..)

Reply:
That is much more sensible. I updated it